README.md

```diff
@@ -30,7 +30,15 @@ const env = $.env.concat([Integer, NonZeroInteger]);
 The next step is to define a `def` function for the environment:
 
 ```javascript
-const def = $.create(env);
+const def = $.create(true, env);
+```
+
+The first argument to `$.create` determines whether type checking is enabled.
+One may elect to pay the performance cost of run-time type checking only during
+development. For example:
```

Comment:
It's possible to misread the second sentence:

_**Only during development** may one elect to pay the performance cost of run-time type checking._

:100: points to the first person to suggest an unambiguous alternative. :)